DataStructures/Graphs/FloydWarshall.java

```diff
@@ -14,7 +14,11 @@ public FloydWarshall(int numberofvertices) {
             [numberofvertices
                 + 1]; // stores the value of distance from all the possible path form the source
     // vertex to destination vertex
-    Arrays.fill(DistanceMatrix, 0);
+    for(int i = 0;i < numberofvertices + 1; i++) {
+      for(int j = 0;j < numberofvertices + 1; j++) {
+        DistanceMatrix[i][j] = 0;
+      }
+    }
```

Comment:
would you please explain benefit of using loops over `Array.fill()`?
i think the previous version of code was good.

Reply:
Well, as I commented below, for loops are much simpler, and "transparent" to the coder, hence chances of mistakes reduce, although if you calculate the time taken, a for loop is likely to make more time than Arrays.fill(). To make use of Arrays.fill() and find out why Arrays.fill() was raising issues, please go through the actual implementation of this on the Oracle Docs.